audiofile/audiofile.h

```diff
@@ -26,12 +26,12 @@ class AudioFile {
             fltp
             };
       SF_INFO info;
-      SNDFILE* sf;
+      SNDFILE* sf{nullptr};
```

Comment:
With brace initialization we usually use spaces around braces, should they perhaps be used here as well? See,for example `musescore.h`:
https://github.com/musescore/MuseScore/blob/742e06641058e2e572bdfab6c80c145daa69f85c/mscore/musescore.h#L255

Reply:
my built-in `lint` stumbled across that too ;-)

Reply:
In my first PR for MuseScore I was asked to remove all brace initialization, even though IMO it's better in several ways. Is this a reversal of this policy against brace initialization?

Reply:
I still fail to see how these are better

Reply:
This is a lack of consistency :)
But in fact we have a lot of places where we use brace initialization of class members, and as far as I remember in your pull request you were asked to remove it from "usual" local variables initialization which is indeed the case when we usually use the old-style `=` initialization.

Reply:
Do you suggest to do this for all types?  
instead  
```
class AudioFile {
      SNDFILE* sf{nullptr};
      bool hasInstrument{false};
      int idx{0};
      FormatType _type{fltp};
      }
```
to do so? 
```
class AudioFile {
      SNDFILE* sf { nullptr };
      bool hasInstrument { false };
      int idx { 0 };
      FormatType _type { fltp };
      }
```

Reply:
yes. Or just `=`

Reply:
Fixed (but I think it's not worth it)

Reply:
With such indentation, initialization becomes similar to single-line methods, additional efforts are needed to understand where the method is and where initialization is - I think this is a problem :)

![image](https://user-images.githubusercontent.com/3818029/77146809-98d4ac00-6a94-11ea-8a74-8739f8111e0b.png)

Reply:
But isn't that a purpose of brace initialization, to make it look uniformly with functions declarations? If this is not desirable one can always use good old `=` initialization.

But this probably belongs to a more general discussion on coding style.

Reply:
> look uniformly with functions declarations?  

I think not, as a minimum for myself I want to clearly distinguish between initialization and functions, otherwise it seems that they forgot to add a `return` and there will be a crash :)

> use good old = initialization.  

He also confuses me, as if I see the body of some method in which they declare  and assign a value.
Especially it can be a problem of perception in large non-trivial classes and writing implementations in .h files - both are practiced in MuseScore :)